vision_agent/lmm/lmm.py

```diff
@@ -6,100 +6,52 @@
 from pathlib import Path
 from typing import Any, Callable, Dict, List, Optional, Union, cast
 
-import requests
 from openai import AzureOpenAI, OpenAI
 
+import vision_agent.tools as T
 from vision_agent.tools.prompts import CHOOSE_PARAMS, SYSTEM_PROMPT
 
 _LOGGER = logging.getLogger(__name__)
 
-_LLAVA_ENDPOINT = "https://svtswgdnleslqcsjvilau4p6u40jwrkn.lambda-url.us-east-2.on.aws"
-
 
 def encode_image(image: Union[str, Path]) -> str:
     with open(image, "rb") as f:
         encoded_image = base64.b64encode(f.read()).decode("utf-8")
     return encoded_image
 
 
+TextOrImage = Union[str, List[Union[str, Path]]]
+MediaChatItem = Dict[str, TextOrImage]
```

Comment:
if we can unify LLM and LMM into one, i think we could also to rename it as `ChatMessage` or just `Message` and make both llm and lmm models take in this type.
I think `List[ChatMessage]` is clearer than `List[MediaChatItem]`.

btw, langchain uses `SystemMessage`, `HumanMessage ` to name these kind of data structure.
so include the word `Message` in the name makes it easier to understand for the bigger langchain community.

Reply:
Makes sense, added the change